### DISCUSSION TITLE
German turn-by-turn announcements

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#54] [Q&A] [answered]
Hello Emux,
you and I know that to be a perfectionist isn't easy all the time ...

With 3.0.9 there was a step back regarding the annoucements in German. Until 3.0.8 everything was fine, but then we got the incorrect German text-to-speech back. Could you please change it again?

All the details from an old thread:

I know that it is only a marginal improvement of the turn-by-turn announcements and I hope that nobody thinks I am a bean counter now. However, I have not yet gotten used to the instructions coming in incorrect German for more than 3 years. I would be interested to know if others feel the same way! By the way, GoogleMaps and Calimoto do it right …

The point is that “Meter” is declined in the dative (by “in …”) and in the plural to “Meter**n**”. For my opinion only a few text strings passed to the TTS would have to be changed:

instead of “in xxx m” → “in xxx Meter**n**”
instead of “in x km” —> “in x Kilometer**n**”

Thank you for a sympathetic review of my request.

Here the thread from 2021: (https://forum.kurviger.de/t/app-text-to-speech-processing/6449)

**Comment by devemux86** [comment, marked as answer]
Hi @WolfgangZ42,

There is no problem! :slightly_smiling_face: 
**Please always report anything you find so we can improve it.**

What do you have selected in "Settings | Voice guidance | Advanced | Text-to-speech"?
(or search settings for "speech")

- "**Normal**" mode uses unit abbreviations (m, km, ft, mi, ...) and text-to-speech engine (Google, Samsung) pronounces the full units
- "**Advanced**" mode post processes turn instructions with full words / language for the units, so they should be announced properly

Looking at the German translation of the units, they are already "Metern" and "Kilometern".

Usually Google text-to-speech works well and pronounces "2 km" as "2 kilometers".
I remember we introduced "Advanced" mode in the past for Samsung text-to-speech.

**Reply by WolfgangZ42** [reply]
Sorry Emux,
this was a false report - I must have thought in my modesty recently TTS "normal" is sufficient for me. Now that I drive again with "Advanced" everything is fine! Thanks for the hint.

**Reply by devemux86** [reply]
Maybe we should convert this setting to a checkbox, so it can have a summary with a description of the "Advanced" mode?

Similarly for the "Settings | Navigation | Advanced | Route appearance", users won't know what "Advanced" means.

Or just change the words "Normal" + "Advanced" to short descriptions.
